app/views/ReasonableExcuseSelectionPage.scala.html

```diff
@@ -24,17 +24,17 @@
         link: components.link
 )
 
-@(form: Form[_], radioOptions: Seq[RadioItem])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(form: Form[_], radioOptions: Seq[RadioItem])(implicit request: UserRequest[_], messages: Messages, appConfig: AppConfig)
 
-@layout(pageTitle = messages("reasonableExcuses.title"),
+@layout(pageTitle = if(request.isAgent){messages("agent.reasonableExcuses.title")}else{messages("reasonableExcuses.title")},
```

Comment:
The heading `agent.reasonableExcuses.title` should only be present when the user is an agent and it is a LSP - according to the comments on PRM-99